services/src/messaging/src/main/java/org/openkilda/messaging/command/discovery/DiscoverIslCommandData.java

```diff
@@ -18,20 +18,13 @@
 import org.openkilda.messaging.command.CommandData;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.fasterxml.jackson.annotation.JsonPropertyOrder;
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.Value;
 
 /**
  * Defines the payload payload of a Message representing an command for ISL discovery.
  */
-@JsonSerialize
-@JsonInclude(JsonInclude.Include.NON_NULL)
-@JsonPropertyOrder({
-        "command",
-        "switch_id",
-        "port_no"})
+@Value
 public class DiscoverIslCommandData extends CommandData {
```

Comment:
You are using NetworkEndpoint as property inside DiscoPacketSendingConfirmation, why not use same approach here?